index.bs

```diff
@@ -881,6 +875,7 @@ execution.setOutput('output', outputBuffer);
 // Start the asynchronous computation.
 await execution.startCompute();
 // The computed result is now in outputBuffer.
+console.log(outputBuffer);
```

Comment:
Curious, how is multiple outputs per node handled?

Reply:
The user code needs to define the multiple outputs when creating a model, e.g.
```js
const model = await nn.createModel(
    [{name: 'output1', operand: output1},
     {name: 'output2', operand: output2}]);
```
Then user code can set the output buffers when executing, e.g.
```js
execution.setOutput('output1', outputBuffer1);
execution.setOutput('output2', outputBuffer2);
```

After the promise returned from `startCompute` solved, the results will be in the outputBuffer1 and outputBuffer2.

Reply:
For the operations, all of the existing operation produce only one operand as the method returned value, e.g.
```js
Operand add(Operand a, Operand b);
```

Later, if we add an operation with multiple outputs, we may need to extend it. @pyu10055, do you know any examples of them?

Reply:
The `gru` operation in [RNNoise and RSNet](https://github.com/webmachinelearning/webnn/pull/82) would have multiple outputs. Let's figure out the API shape when adding it to the spec.